stream_alert_cli/terraform/generate.py

```diff
@@ -486,9 +486,60 @@ def terraform_generate_handler(config, init=False, check_tf=True, check_creds=Tr
         message='Removing old Alert Merger Terraform file'
     )
 
+    # Setup Lookup Tables if applicable
+    _generate_lookup_tables_settings(config)
+
     return True
 
 
+def _generate_lookup_tables_settings(config):
+    """
+    Generates .tf.json file for LookupTables
+    """
+    tf_file_name = 'terraform/lookup_tables.tf.json'
+
+    if not config['lookup_tables'].get('enabled', False):
+        remove_temp_terraform_file(tf_file_name, 'Removing old LookupTables Terraform file')
+        return
+
+    dynamodb_tables = []
+    s3_buckets = []
+
+    for _, table_config in config['lookup_tables'].get('tables', {}).items():
+        if table_config['driver'] == 's3':
+            s3_buckets.append(table_config['bucket'])
+            continue
+
+        if table_config['driver'] == 'dynamodb':
+            dynamodb_tables.append(table_config['table'])
+            continue
+
+    roles = [
+        '${module.alert_processor_lambda.role_id}',
+        '${module.alert_merger_lambda.role_id}',
+        '${module.rules_engine_lambda.role_id}',
+    ]
+
+    for cluster in config.clusters():
+        roles.append('${{module.classifier_{}_lambda.role_id}}'.format(cluster))
+
+    generated_config = {
+        'module': {
+            'lookup_tables_iam': {
+                'source': 'modules/tf_lookup_tables',
+                'account_id': config['global']['account']['aws_account_id'],
+                'region': config['global']['account']['region'],
+                'prefix': config['global']['account']['prefix'],
+                'dynamodb_tables': dynamodb_tables,
+                's3_buckets': s3_buckets,
+                'roles': roles,
+            }
+        }
+    }
+    with open(tf_file_name, 'w') as tf_file:
```

Comment:
note to self: we should refactor this tf generation crap so there is a central facility for this logic.. instead of a bunch of `json.dump(...)` nonsense lying around

Reply:
lol no need to address now. I was just confused reviewing this as to why you had to do the writing out of the file within this function.. then realize _every_ generation function is like this